replica/database.hh

```diff
@@ -441,8 +441,7 @@ private:
 
     uint64_t _failed_counter_applies_to_memtable = 0;
 
```

Comment:
> This patch unfreezes all frozen mutations at the
(async) table::apply layer where we can yield if needed
and then eliminates the memtable apply path, given

Likely to regress performance, on top of the coroutinization.